src/CovidDataDashboard/CovidDataClip.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FunctionComponent } from "react"
+import { ActivityIndicator, StyleSheet, View } from "react-native"
+import { useTranslation } from "react-i18next"
+
+import CovidDataStateTrend from "./CovidDataStateTrend"
+import { Text } from "../components"
+import { useConfigurationContext } from "../ConfigurationContext"
+import {
+  CovidDataRequestStatus,
+  useCovidDataContext,
+} from "../CovidDataContext"
+
+import {
+  Affordances,
+  Colors,
+  Iconography,
+  Spacing,
+  Typography,
+} from "../styles"
+
+const LoadingIndicator = () => {
+  return (
+    <View style={style.activityIndicatorContainer}>
+      <ActivityIndicator
+        size={"large"}
+        color={Colors.neutral100}
+        style={style.activityIndicator}
+        testID={"loading-indicator"}
+      />
+    </View>
+  )
+}
+
+const CovidDataClip: FunctionComponent = () => {
+  const { stateAbbreviation } = useConfigurationContext()
+  const { t } = useTranslation()
+  const {
+    covidDataRequest: { data, status },
+  } = useCovidDataContext()
+
+  if (stateAbbreviation === null) {
+    return null
+  }
+
+  const dataClipContent = () => {
+    switch (status) {
+      case CovidDataRequestStatus.LOADING:
+        return <LoadingIndicator />
+      case CovidDataRequestStatus.ERROR:
+        return (
+          <Text style={style.errorMessageText}>
+            {t("covid_data.error_getting_data", {
+              location: stateAbbreviation,
+            })}
+          </Text>
+        )
+      case CovidDataRequestStatus.SUCCESS:
+        return (
+          <CovidDataStateTrend
+            lastWeekData={data}
+            stateAbbreviation={stateAbbreviation}
+          />
+        )
+      case CovidDataRequestStatus.NOT_STARTED:
+        return (
+          <Text style={style.errorMessageText}>
```

Comment:
Why do we return the error message in the `NOT_STARTED` state?

Reply:
I was not sure about it, we probably should never reach this point, since this component shouldn't be rendered with this state but typescript is complaining that it is not complete and I liked not having a default case and have the type system keeping me honest. Maybe the name of the status is confusing?

Reply:
I think it's fine for now!